cadence/contracts/Giefts.cdc

```diff
@@ -72,6 +72,8 @@ pub contract Giefts {
         /// A collection of NFTs
         /// nfts are stored as a map of uuids to NFTs
         access(contract) var nfts: @{UInt64: NonFungibleToken.NFT}
+        /// Registry capability
+        access(contract) let registryCapabilty: Capability<&FindRegistry.Registry{FindRegistry.RegistryPublic, FindRegistry.RegistryPrivate}>?
```

Comment:
Not sure we can add a field here inside of a resource.

Reply:
oh yeah I forgot about that, new contract?

Reply:
Yeah new contract GiefMe.cdc and we remove the ability to create new Giefts on this one.